_posts/2022-01-21-dashboards-plugins-intro.markdown

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: "Introduction to OpenSearch Dashboard Plugins"
+authors:
+  - ashwinpc
+date: 2022-01-21
```

Comment:
Do you really want this on the 21st? Is that date important?

Reply:
No it's not. Feel free to pick any other date. I just picked this to make sure we had enough time to review the changes and merge it in since i didnt know how long that usually took